### PR TITLE
Patch legacy article 15

### PR DESCRIPTION
**Pull request by manwar**
Hi @briandfoy 

Please review the PR.

Many Thanks
Best Regards,
Mohammad S Anwar

### CODE REVIEW

File: content/legacy/_pub_2004_08_27_bricolage.md [hunk 1]
```diff
@@ -120,12 +120,12 @@ Bricolage templates correspond to document categories and elements, so that one
...
+    CPAN"), [XML](https://metacpan.org/pod/XML::Generator "XML::Generator on CPAN"), [PDFs](https://metacpan.org/pod/PDFLib "PDFLib on
+    CPAN"), or even [Excel files](https://metacpan.org/pod/Spreadsheet::WriteExcel/ "Spreadsheet::WriteExcel on CPAN")!
 
 Furthermore, Bricolage has a sub-classable templating architecture, which means that, with a little bit of work, you can add your own templating architecture. Anyone care to add support for [Embperl](http://perl.apache.org/embperl/ "Embperl home"), [Apache::ASP](http://www.apache-asp.org/ "Apache::ASP
-    home"), or [XSLT](http://search.cpan.org/dist/XML-LibXSLT/ "XML::LibXSLT on
+    home"), or [XSLT](https://metacpan.org/pod/XML::LibXSLT "XML::LibXSLT on
```
Comment (dnmfarrell) [inline content/legacy/_pub_2004_08_27_bricolage.md]: This link looks broken - should the text in double quotes be the link label text?

**Comment by dnmfarrell** [comment]
Hey @manwar 

Thanks for this, it looks good except for the one comment I posted. Let me know what you think.

**Comment by manwar** [comment]
Hi @dnmfarrell 

Please review again.

Many Thanks.
Best Regards,
Mohammad S Anwar

**Comment by briandfoy** [comment]
I fixed up the Bricolage article. There were many problems in it besides the MetaCPAN link. That style was throughout the article. It's in 6d3bb88ce161368a65d5d14c3d5198140cbe6ec8.

**Pull request merged**